setup.py

```diff
@@ -48,6 +46,7 @@ def read(*rnames):
           ],
           'test': [
               'pytest',
+              'six',
```

Comment:
This would allow six only in the tests. When we start to use it outside the tests we will not see this missing install dependency as the tests always use the `test` extra. So please move `six` back to the install requirements.